tailwind.config.cjs

```diff
@@ -101,6 +101,7 @@ export default {
             fontFamily: {
                 display: ["Fabriga", "serif"],
                 sans: ["Inter", "sans-serif"],
+                mono: ["ui-monospace", "SFMono-Regular", "Menlo", "Monaco", "Consolas", "Liberation Mono", "Courier New", "monospace"]
```

Comment:
is that relevant for the changes we need to make for inline link styling?

Reply:
this is not relevant for the inline-link class but I needed it to display the various classNames as "code" in our storybook overview file. It’s more or less the default value from tailwind setting the default/system-fonts-fallback fonts :-)

Reply:
as of right now we don’t use `font-mono` anywhere so I shouldn’t break something

Reply:
thanks for throwing light on! I would prefer to not mix things up and treat it as two different implementation to guarantee isolated changes. Since this is not used it is ok for now.